### PR TITLE
Add TopH and BottomH for only controlling height of the window.

### PR DESCRIPTION
**Pull request by exorcist365**
Yeah, the title says it all, I have been using it for the past week and had no issues except for #555 but that happens when the position is set to static {} as well.

### CODE REVIEW

**Comment by jaor** [comment]
this looks good, thank you.  could you please mention the new options
in the doc?

**Comment by exorcist365** [comment]
Yeah, sure.

File: src/Xmobar/X11/Window.hs [hunk 1]
```diff
@@ -80,9 +80,11 @@ setPosition c p rs ht =
   case p' of
     Top -> Rectangle rx ry rw h
     TopP l r -> Rectangle (rx + fi l) ry (rw - fi l - fi r) h
+    TopH ch -> Rectangle rx ry rw (fi ch)
```
Comment (liskin) [inline src/Xmobar/X11/Window.hs:83]: I wonder why this uses `ch` directly but `TopSize` uses `mh ch`. Is that useful in practice?

Comment (exorcist365) [reply src/Xmobar/X11/Window.hs:83]: Do I make it use mh ch or make TopSize use just ch?

Comment (jaor) [reply src/Xmobar/X11/Window.hs:83]: in theory, mh ch would be more correct, although i think it should never happen that they differ in this case.  it think it's better if we don't change TopSize.

btw, @exorcist365, thanks for the docs! would you mind squashing all your changes to a single commit (now it looks like a collection of 4)?

Comment (exorcist365) [reply src/Xmobar/X11/Window.hs]: Yeah, done.

**Review by jaor — COMMENTED** [review]
merged, thanks

**Pull request merged**